web_external/models/UserModel.js

```diff
@@ -1,11 +1,12 @@
 isic.models.UserModel = girder.models.UserModel.extend({
     name: function () {
-        var name;
+        var name = this.get('name');
```

Comment:
This exposes a bug that already existed in some form. The `name` attribute isn't generally defined for users. Right now on the live site I see "undefined [Max Smolens (msmolens)]" in the header user dropdown.

Perhaps this could return null or an empty string if there is no `login` attribute?